tools/spatyper/spatyper.xml

```diff
@@ -0,0 +1,76 @@
+<tool id="spatyper" name="Generate spa type identification" version="@VERSION@+galaxy0" profile="21.01">
+    <macros>
+        <token name="@VERSION@">0.3.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">spatyper</requirement>
+    </requirements>
+    <command detect_errors="aggressive"> <![CDATA[
+        spaTyper -f '${input1}' -r '${input2}' > '${output1}' 
+    ]]> </command>
+    <inputs>
+        <param type="data" name="input1" format="fasta" label="SPA Fasta file" help="Staphylococcus aureus - staphylococcal protein A fasta file. This will be assigned a type based on its repeat region."/>
+        <param type="data" name="input2" format="fasta" label="Repeats Library Multifasta" help="A multifasta representing a set of known repeat regions found in staphylococcal protein A. This can be downloaded at (http://spa.ridom.de/dynamic/sparepeats.fasta)"/>
+    </inputs>
+    <outputs>
+        <data name="output1" format="txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="input.fasta"/>
+            <param name="input2" value="sparepeats.fasta"/>
+            <output name="output1">
+                <assert_contents>
+                    <has_text text = "t11215" />
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+::
+    USAGE: spaTyper [-h] [-r REPEAT_FILE] [-o REPEAT_ORDER_FILE] [-d FOLDER] [-f FASTA [FASTA ...]] [-g GLOB] [-e] [--output OUTPUT] [--version] [--debug] [--info]
+
+    spaTyper.py: Get spa types
+
+    Version: 0.3.3
+    License: GPLv3
+    Prints spa type to stdout
+
+    It will download sparepeats.fasta and spatypes.txt to repository directory 
+    if files not provided or already in directory. It can be loaded as a python
+    module. Python 3 version only.
+
+    Options: -h     show this help message and exit
+             -r     REPEAT_FILE (http://spa.ridom.de/dynamic/sparepeats.fasta)
+             -o     REPEAT_ORDER_FILE (http://spa.ridom.de/dynamic/spatypes.txt)
+             -d     Folder to save downloaded files from Ridom/Spa server
+             -f     List of one or more fasta files.
+             -g     Uses unix style pathname expansion to run spa typing on all files. If your shell autoexpands wildcards use -f.
+             -e     Do PCR product enrichment. [Default: False]
+
+    Example Output:
+
+    Start the identification of repeats in Spa protein:
+    + Repeats fasta file provided:  ~/tools/spaTyper/test-data/sparepeats.fasta
+    + Check or download repeats types file in folder:  ~/planemo/bin
+        A previous download created spatypes.txt on: 2023-03-15 17:43:58
```

Comment:
I wonder if `planemo l` is complaining about the additional indentation in this line?

```
Applying linter help... WARNING
.. WARNING: Invalid reStructuredText found in help - [<string>:28: (ERROR/3) Unexpected indentation.
].
```

It was showing the help text before, so maybe just remove the entire `Example Output:` section since it doesn't seem to work in restructured text right now? Or, you could test out by changing the indentation.

![image](https://user-images.githubusercontent.com/200517/226437645-542d4bcd-9327-433e-832c-40ca960acca1.png)